ansible_collections/arista/avd/roles/eos_designs/templates/overlay/ibgp/peer-groups.j2

```diff
@@ -29,8 +28,8 @@
       update_source: Loopback0
       remote_as: "{{ bgp_as }}"
       bfd: true
-{%     if bgp_peer_groups.rr_overlay_peers.password is arista.avd.defined() %}
-      password: "{{ bgp_peer_groups.rr_overlay_peers.password }}"
+{%     if switch.bgp_peer_groups.rr_overlay_peers.password is arista.avd.defined() %}
+      password: "{{ switch.bgp_peer_groups.rr_overlay_peers.password }}"
       send_community: all
       maximum_routes: 0
```

Comment:
Not related to this PR. But lines 20,21 are producing the same output as lines 33,34.